csharp-ovh/Client/QueryStringParams.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Web;
+
+namespace Ovh.Api
+{
+    public class QueryStringParams : List<(string, string)>
+    {
+        private List<string> _existingKeys = new List<string>();
+
+        public void Add(string key, string value)
+        {
+            if(_existingKeys.Contains(key))
+            {
+                throw new ArgumentException("Duplicate keyword. This is currently not supported by OVH API");
+            }
+            Add((key, value));
+        }
+
+        public string ToQueryString(bool includeQuestionMark = true)
```

Comment:
Why don't you override `.ToString()`?

Reply:
1. At first, this was an extension method, so historically it made sense
2. .ToString() doesn't take a `bool` param. However I'm pretty sure I won't ever use that with `false`. Yet, a real query string does not contain it.

=> I'll replace that with `.ToString()`